tests/test_modelscan.py

```diff
@@ -243,6 +252,13 @@ def test_scan_zip(zip_file_path: Any) -> None:
     assert ms.issues.all_issues == expected
 
 
+def test_scan_pytorch(pytorch_file_path: Any) -> None:
+    bad_pytorch = Modelscan()
+    bad_pytorch.scan_path(Path(f"{pytorch_file_path}/bad_pytorch.pt"))
+    assert bad_pytorch.issues.all_issues == []
+    assert [error.scan_name for error in bad_pytorch.errors] == ["pytorch"]  # type: ignore[attr-defined]
```

Comment:
Not relevant to this PR. scan_name need to be defined as an attribute of the base Error class to avoid the type ignore. I will take care of it in a new PR.